static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -350,11 +346,8 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
     );
   }
 
-  if (
-    !currentPlatform ||
-    (!newOnboarding && !hasOnboardingContents) ||
-    (newOnboarding && !newDocs)
-  ) {
+  // No platform or no docs (new nor old)
+  if (!currentPlatform || (!newDocs && !hasOnboardingContents)) {
```

Comment:
Can you help me confirm this, i boiled it down to `if (!currentPlatform || !newDocs) {` but it's late and i might've messed it up

Reply:
this is my check for both old & new docs, because the chunk below renders new docs if they're available, and old docs if no new docs. so if there aren't any old docs either (for whatever reason) we don't want to hit that code and should render "fiddlesticks" instead. does that make sense?

Reply:
idk for sure if we should fallback to old docs... they might be really old & stale. I believe that all the docs which could appear inside this replay sidebar are in the new style, so it's probably simpler to log into sentry if something unexpected happens, and then we can be deleting the old docs safely.